staging/src/k8s.io/apiserver/pkg/server/openapi/openapi.go

```diff
@@ -107,6 +157,40 @@ func (o *openAPI) init(webServices []*restful.WebService) error {
 			return err
 		}
 	}
+	if err := o.makeAdditionalFormats(); err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func (o *openAPI) makeAdditionalFormats() error {
```

Comment:
Docstring on this would be nice. I can't figure out the purpose of this function.